packages/react-storage/src/components/StorageBrowser/context/elements/definitions.ts

```diff
@@ -164,6 +165,13 @@ export const HeadingElement = defineBaseElement({
   displayName: 'Title',
 });
 
+export interface Heading3Element3Props
```

Comment:
`Heading3Element3Props` is missing a `3` 😆

Reply:
fixed

Reply:
33333333 @reesscot how you missed it ????????